.travis.yml

```diff
@@ -3,7 +3,7 @@ language: node_js
 node_js:
   # we recommend testing addons with the same minimum supported node version as Ember CLI
   # so that your addon works for all apps
-  - "8"
+  - "10"
```

Comment:
is Node 8 no longer supported? is this a breaking change?

Reply:
the latest fibers isn't supported by node 8 (it needs at least node 10). There is a dev dep that depends on the latest version of fibers (which I have considered rolling back just for this since it's my package).

This doesn't impact users of this as a dependency.

Reply:
that's true, but if we're no longer testing on Node 8 we might have other incompatible deps creep in 😉

Reply:
this is true, which is why I was considering reverting away from the latest fibers version for the dependency. However, with the current version being in a kind of "maintenance mode" and node 8's end of life being only in 5 months, I thought that the risk was manageable.

Reply:
if only one of those was true, I'd be much more hesitant. With both of them, I feel comfortable with it.